docs/application/native/guides/compliations/watchface-complication.md

```diff
@@ -128,20 +134,43 @@ To get the complication provider data, the following functions can be used:
 | `watchface_complication_data_get_long_text()` | long text |
 | `watchface_complication_data_get_title()` | title text |
 | `watchface_complication_data_get_timestamp()` | timestamp |
+| `watchface_complication_data_get_timeinfo()` | time information |
+| `watchface_complication_timeinfo_get_timezone()` | timezone |
+| `watchface_complication_timeinfo_get_timezone_id()` | timezone ID |
+| `watchface_complication_timeinfo_get_timezone_country()` | timezone country |
+| `watchface_complication_timeinfo_get_timezone_city()` | timezone city |
+| `watchface_complication_timeinfo_get_timestamp()` | timestamp |
+| `watchface_complication_timeinfo_destroy()` | Destroys time information |
 | `watchface_complication_data_get_image_path()` | image path |
 | `watchface_complication_data_get_ranged_value()` | min / max / current value |
 | `watchface_complication_data_get_icon_path()` | icon path |
 | `watchface_complication_data_get_extra_data()` | extra data |
 
+
 ```cpp
 void _watchface_complication_updated_cb(int complication_id,

```

Comment:
I've found the indentations in this sample code is tabs.
Would you please replace tabs with 4 spaces?